project/Dependencies.scala

```diff
@@ -2,5 +2,5 @@ import sbt._
 
 object Dependencies {
   lazy val SparkSql = "org.apache.spark" %% "spark-sql" % "2.2.0"
-  lazy val SourcedEngine = "tech.sourced" % "engine" % "0.4.1"
+  lazy val SourcedEngine = "tech.sourced" % "engine" % "0.5.1"
```

Comment:
@smacker Thanks! Could you update the version directly to `0.5.2`?

Reply:
done. Hopefully 0.5.3 won't be released before merge 😆